src/PlusDataCollection/StealthLink/vtkPlusStealthLinkTracker.cxx

```diff
@@ -917,10 +987,36 @@ PlusStatus vtkPlusStealthLinkTracker::InternalUpdate()
         this->ToolTimeStampedUpdateWithoutFiltering(toolIterator->second->GetSourceId(), instrumentToTrackerTransform, TOOL_OUT_OF_VIEW, unfilteredTime, timeSystemSec);
       }
     }
+    // if the wanted Transform is any tool to Tracker that is not considered the current instrument
     else
     {
       vtkSmartPointer<vtkMatrix4x4> transformMatrixForNotTrackedTool = vtkSmartPointer<vtkMatrix4x4>::New();
-      this->ToolTimeStampedUpdateWithoutFiltering(toolIterator->second->GetSourceId(), transformMatrixForNotTrackedTool, TOOL_OUT_OF_VIEW, unfilteredTime, timeSystemSec);
+      // if we are tracking the current instrument at all times

```

Comment:
Write here a bit more about what is the current instrument (closest to the reference?), why we don't track all instruments by default (due to time it takes for the query to compete), ...